lib/tasks/boostrap.rake

```diff
@@ -21,4 +23,20 @@ namespace :db do
     u.save
     puts "Created user #{u.email} with password #{u.password}"
   end
+
+  desc 'Create non-moderator'
+  task bootstrap: %i(protect environment) do
```

Comment:
We might need to change the task name because the same name was used for `Create dummy user`. 
I would suggest that admin and non-admin users will be created in the same `bootstrap` task